pkg/operator/etcdendpointscontroller/etcdendpointscontroller.go

```diff
@@ -89,13 +92,19 @@ func (c *EtcdEndpointsController) syncConfigMap(recorder events.Recorder) error
 		return fmt.Errorf("couldn't determine bootstrap status: %w", err)
 	}
 
+	etcdMembers, err := c.etcdClient.MemberList()
+	if err != nil {
+		return fmt.Errorf("could not create etcd client: %w", err)
+	}
+	memberHealth := etcdcli.GetMemberHealth(etcdMembers)
```

Comment:
this cannot fail?

Reply:
added test for empty []healthCheck but we should not have a failure from nil pointer here. When I added the mock etcd servers to `TestBootstrapAnnotationRemoval` it does test the absolute minimum response configuration else err.